shell/prompt.go

```diff
@@ -55,3 +55,16 @@ func PromptUserForYesNo(prompt string, terragruntOptions *options.TerragruntOpti
 		return false, nil
 	}
 }
+
+func PrintErrorOutput(prompt string, terragruntOptions *options.TerragruntOptions) error {
+	n, err := terragruntOptions.ErrWriter.Write([]byte(prompt))
+	if err != nil {
+		terragruntOptions.Logger.Error(err)
+		return errors.WithStackTrace(err)
+	}
+	if n != len(prompt) {
+		terragruntOptions.Logger.Errorln("Failed to write data")
+		return errors.WithStackTrace(err)
+	}
+	return nil
+}
```

Comment:
Is this function needed? Can't this simplify to `terragruntOptions.ErrWriter.Write([]byte(prompt))`?

The reason I think so is that the second check for `n != len(prompt)` is dead code because of the spec of `io.Writer.Write` (from https://github.com/golang/go/blob/1537f14/src/io/io.go#L87-L94):

> // Write must return a non-nil error if it returns n < len(p)